language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/SubmitCommandsRequest.java

```diff
@@ -323,18 +328,7 @@ private static CommandsOuterClass.Commands toProto(
 
   public static CommandsOuterClass.Commands toProto(
       @NonNull String ledgerId, @NonNull CommandsSubmission submission) {
-    return toProto(
-        ledgerId,
-        submission.getWorkflowId().orElse(""),
-        submission.getApplicationId(),
-        submission.getCommandId(),
-        submission.getActAs(),
-        submission.getReadAs(),
-        submission.getMinLedgerTimeAbs(),
-        submission.getMinLedgerTimeRel(),
-        submission.getDeduplicationTime(),
-        Optional.empty(),
-        toCommands(submission.getCommands()));
+    return toProto(ledgerId, submission.getWorkflowId(), submission);
```

Comment:
This one does not need to use the deprecated `toProto` implementation, so it will be able to propagate the disclosed contracts.